src/promptflow/promptflow/executor/flow_executor.py

```diff
@@ -957,6 +946,13 @@ def _extract_outputs(self, nodes_outputs, bypassed_nodes, flow_inputs):
                     output_name=name,
                     node_name=node.name,
                 )
+            if output.reference.value in bypassed_nodes:
+                logger.warning(
+                    msg=(
+                        f"The node referenced by output:'{output.reference.value}' is bypassed, \
+                            which is not recommended. "
```

Comment:
Please use `flow_logger` to print this message because these logs will also be printed in batch run mode and it may disrupt the useful information printed by the `bulk_logger`.

```suggestion
flow_logger.warning(f"The node referenced by output:'{output.reference.value}' is bypassed, which is not recommended.")
```

Reply:
Changed logger to flow_logger.